openapi.yaml

```diff
@@ -347,71 +361,131 @@ paths:
         "404":
           description: "Person not found"
 
+  /person/{id}/getAllPassports:
```

Comment:
here I replace  /passport endpoint to make this functionality to auth user, but Ithink that function should be able only for Admin Role
P.S. maybe this endpoint will be needed later to our API

Reply:
It seems pretty much similar to /searches endpoint, just another parameters there

Reply:
it moved to search group, and fixed endpoint